package.json

```diff
@@ -29,5 +29,10 @@
     "concurrently": "^7.6.0",
     "cross-env": "^7.0.3",
     "husky": "^8.0.3"
+  },
+  "dependencies": {
+    "axios": "^1.3.5",
+    "dayjs": "^1.11.7",
+    "react-reveal": "^1.2.2"
```

Comment:
axios, dayjs, react-reveal is frontend package. So they must be in the frontend package